src/include/storage/store/rel_table_data.h

```diff
@@ -61,6 +61,7 @@ struct CSRHeaderColumns {
 // TODO(Guodong): Serialize the info to disk. This should be a config per node group.
 struct PackedCSRInfo {
     uint64_t calibratorTreeHeight;
+    // TODO(Guodong): Should this be removed?
```

Comment:
Will clean this up.

Reply:
OK, will keep these here: https://github.com/kuzudb/kuzu/issues/3729